recipes/colored/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "colored" %}
+{% set version = "1.3.93" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8296ea990e3f6b7822f44eec21408b126dfb9c1c031306b859e3f7d46cc27075
+
+build:
+  number: 0
```

Comment:
Please also add `noarch: python` as the builds of this package don't depend on the Python version.